### PR TITLE
feat: able to get depository certificate with gopdf

### PR DESCRIPTION
**Pull request by bjwswang**
Fix: #45 

## PR Summary

- add a new fiber handler to generate/download depository certificate with [pre-defined certificate template](https://github.com/bestchains/bc-saas/pull/46/commits/ac0b8d382b81f070ea571e1687dd29f2908ffe4a#diff-5f9c2d984945931f7416dfcc010da701de50de6e1b65b1b8894d52ca7486ca2e)

- add resource files which are required by gopdf under [resource](https://github.com/bestchains/bc-saas/pull/46/commits/ac0b8d382b81f070ea571e1687dd29f2908ffe4a#diff-4450e766ac5541efcd93347a8204f497654c68eb57da6ad9c968bc76ad1d49f7)

- update dockerfile to add [resource files](https://github.com/bestchains/bc-saas/pull/46/commits/ac0b8d382b81f070ea571e1687dd29f2908ffe4a#diff-d882ff29039c4d601d6423977d203ccafae5e1e2dc59f8eef681e732d1534dceR15)

- update [deployment file](https://github.com/bestchains/bc-saas/pull/46/commits/ac0b8d382b81f070ea571e1687dd29f2908ffe4a#diff-1fd39825521949dd2e734810232585533819d7332aa0ca0060518f77bd72e075) so `bc-saas` can use the right `cert template files`

- update the relevant docs

## Test case

Download a depository certificate with the `kid`

Call this url from browser

```
localhost:9999/basic/depositories/certificate/02e853e0f68566e62fddd9c4e014db65b7f315d9
```

A pdf will be downloaded.

[02e853e0f68566e62fddd9c4e014db65b7f315d9.pdf](https://github.com/bestchains/bc-saas/files/11813983/02e853e0f68566e62fddd9c4e014db65b7f315d9.pdf)

### CODE REVIEW

**Comment by bjwswang** [comment]
@dayuy  Response headers
```golang
	ctx.Response().Header.Add("Content-Type", "application/octet-stream")
	ctx.Response().Header.Add("Content-Disposition", fmt.Sprintf("attachment; filename=%s.pdf", kid)
```

**Comment by bjwswang** [comment]
@0xff-dev What else that we should do for the bc-saas deployment?

File: pkg/utils/gopdf.go [hunk 1]
```diff
@@ -80,7 +80,7 @@ const (
 	// defaultTtfFontPath is the default path to ttf font
 	// stsong.ttf downloaded from https://www.wfonts.com/font/stsong
 	// Reason to use this font: it supports Chinese(https://github.com/signintech/gopdf/issues/68)
-	defaultTtfFontPath = "./ttf/stsong.ttf"
+	defaultTtfFontPath = "../../resource/ttf/stsong.ttf"
```
Comment (0xff-dev) [inline pkg/utils/gopdf.go:83]: Maybe `resource/ttf/stsong.ttf`.
If it is `../../resource/ttf/stsong.ttf`, then the program should run in the `ttf` directory to find the relevant resources by default.

Comment (0xff-dev) [reply pkg/utils/gopdf.go:83]: But currently there is no effect on the output of the pdf.

**Comment by 0xff-dev** [comment]
> @0xff-dev What else that we should do for the bc-saas deployment?

I think it's ok to update the image directly.

**Review by 0xff-dev — APPROVED** [review]

**Review by Abirdcfly — APPROVED** [review]

**Pull request merged**